fixtures.php

```diff
@@ -29,4 +29,43 @@ function apply_filters($hook_name, $value, ...$args) {};
 function do_action($hook_name, ...$args) {};
 PHP;
 
+$fixtures['$global']['functions']['wp_send_json'] = <<<'PHP'
+/**
+ * @param mixed $response
+ * @param int|null $status_code
+ * @param int $options
+ * @psalm-return never
+*/
+function wp_send_json($response, $status_code = null, $options = 0) {};
+PHP;
+
+$fixtures['$global']['functions']['wp_send_json_success'] = <<<'PHP'
+/**
+ * @param mixed $data
+ * @param int|null $status_code
+ * @param int $options
+ * @psalm-return never
+*/
+function wp_send_json_success( $data = null, $status_code = null, $options = 0 ) {};
```

Comment:
The existing fixtures use Inpsyde PHP Coding Standards, i.e., no whitespace inside parentheses. Let's align with that.
```suggestion
function wp_send_json_success($data = null, $status_code = null, $options = 0) {};
```